2025-10-05 15:43:30,152 | ERROR | API_KEY or API_SECRET not found in environment. Exiting.
2025-10-05 15:48:40,997 | ERROR | API_KEY or API_SECRET not found in environment. Exiting.
2025-10-05 16:36:46,934 | ERROR | API_KEY or API_SECRET not found in environment. Exiting.
2025-10-05 16:36:55,734 | ERROR | API_KEY or API_SECRET not found in environment. Exiting.
2025-10-05 16:37:00,771 | ERROR | API_KEY or API_SECRET not found in environment. Exiting.
2025-10-05 16:47:38,692 | INFO | Placing MARKET order: BTCUSDT BUY 0.001
2025-10-05 16:47:39,116 | ERROR | Binance API error: APIError(code=-2015): Invalid API-key, IP, or permissions for action
Traceback (most recent call last):
  File "D:\binance_bot\src\market_orders.py", line 71, in place_market_order
    order = client.futures_create_order(
  File "D:\binance_bot\venv\lib\site-packages\binance\client.py", line 7659, in futures_create_order
    return self._request_futures_api("post", "order", True, data=params)
  File "D:\binance_bot\venv\lib\site-packages\binance\client.py", line 126, in _request_futures_api
    return self._request(method, uri, signed, force_params, **kwargs)
  File "D:\binance_bot\venv\lib\site-packages\binance\client.py", line 89, in _request
    return self._handle_response(self.response)
  File "D:\binance_bot\venv\lib\site-packages\binance\client.py", line 98, in _handle_response
    raise BinanceAPIException(response, response.status_code, response.text)
binance.exceptions.BinanceAPIException: APIError(code=-2015): Invalid API-key, IP, or permissions for action
2025-10-05 16:49:04,284 | ERROR | API_KEY or API_SECRET not found in environment. Exiting.
2025-10-05 16:49:14,112 | ERROR | API_KEY or API_SECRET not found in environment. Exiting.
2025-10-05 16:49:24,648 | ERROR | API_KEY or API_SECRET not found in environment. Exiting.
2025-10-05 16:49:49,965 | INFO | Placing MARKET order: BTCUSDT BUY 0.001
2025-10-05 16:49:50,116 | ERROR | Binance API error: APIError(code=-2015): Invalid API-key, IP, or permissions for action
Traceback (most recent call last):
  File "D:\binance_bot\src\market_orders.py", line 71, in place_market_order
    order = client.futures_create_order(
  File "D:\binance_bot\venv\lib\site-packages\binance\client.py", line 7659, in futures_create_order
    return self._request_futures_api("post", "order", True, data=params)
  File "D:\binance_bot\venv\lib\site-packages\binance\client.py", line 126, in _request_futures_api
    return self._request(method, uri, signed, force_params, **kwargs)
  File "D:\binance_bot\venv\lib\site-packages\binance\client.py", line 89, in _request
    return self._handle_response(self.response)
  File "D:\binance_bot\venv\lib\site-packages\binance\client.py", line 98, in _handle_response
    raise BinanceAPIException(response, response.status_code, response.text)
binance.exceptions.BinanceAPIException: APIError(code=-2015): Invalid API-key, IP, or permissions for action
2025-10-05 16:59:18,039 | ERROR | API_KEY or API_SECRET not found in environment. Exiting.
2025-10-05 17:00:09,100 | ERROR | API_KEY or API_SECRET not found in environment. Exiting.
2025-10-05 17:05:06,777 | ERROR | Binance API error
Traceback (most recent call last):
  File "D:\binance_bot\src\binance_client.py", line 37, in place_market_order
    res = self.client.futures_create_order(
  File "D:\binance_bot\venv\lib\site-packages\binance\client.py", line 7659, in futures_create_order
    return self._request_futures_api("post", "order", True, data=params)
  File "D:\binance_bot\venv\lib\site-packages\binance\client.py", line 126, in _request_futures_api
    return self._request(method, uri, signed, force_params, **kwargs)
  File "D:\binance_bot\venv\lib\site-packages\binance\client.py", line 89, in _request
    return self._handle_response(self.response)
  File "D:\binance_bot\venv\lib\site-packages\binance\client.py", line 98, in _handle_response
    raise BinanceAPIException(response, response.status_code, response.text)
binance.exceptions.BinanceAPIException: APIError(code=-2015): Invalid API-key, IP, or permissions for action
